.github/ISSUE_TEMPLATE/suggest_wallet.md

```diff
@@ -6,98 +6,192 @@ labels: "wallet :purse:,"
+
+<!-- Please provide a URL to the documentation -->
+
+**Is your wallet security tested? Please explain security measures i.e. security audit, internal security team, or some other method.**
 
 <!-- Please explain any security measures you have taken to ensure your wallet is secure -->
 
 **When did your wallet go live to users?**
 
-<!-- Please provide an exact or approximate date when your wallet was usable by the public -->
+<!-- Please provide a date when your wallet was usable by the public. Please provide some user metrics for how many users are using this wallet. -->
 
 **Does your wallet have an active development team?**
 
-<!-- Are developers actively working on the wallet? -->
-
-**Is your wallet open-source?**
-
-<!-- If yes, please provide a direct link to the repository, if you can -->
+<!-- Are developers actively working on the wallet? Provide proof that the wallet is actively being worked on (ex. GitHub repo link). -->
 
 **Is your wallet globally accessible?**
 
 <!-- Please list any KYC requirements or geographic limitations of your wallet, if any exist. -->
 
-**Is your wallet custodial, non-custodial, or a hardware wallet?**
+**Is there internationalization support for your wallet?**
+
+<!-- Please list languages that the wallet actively supports -->
+
+**What is the repository for your codebase?**
+
+<!-- If you’re project is open source, please provide a link to the codebase. -->
+
+**What social links are there for the project?**
+
+<!-- Please provide social links for the wallet (Discord, Twitter, etc.) -->
+
+**What is your wallet's onboarding experience?**
+
+<!-- Please provide an explanation of where to find the user onboarding experience (links, is it built into the app, etc.), and what the onboarding experience is. -->
+
+### Device
+
+**Does the wallet have a mobile app? If yes, which operating systems are supported (iOS, Android)?**
+
+<!-- If the wallet has a mobile interface, please provide information and links to the app store for the operating systems supported. -->
+
+**Does the wallet have a desktop app? If yes, which operating systems are supported (Windows, Mac, Linux)?**
+
+<!-- If the wallet has a desktop interface, please provide links and information for the operating systems supported. -->
+
+**Does the wallet have a browser extension? If yes, which browsers are supported (Chromium, Firefox, Safari)?**
+
+<!-- If the wallet has a browser extension, please provide links and information for the browsers that are supported. -->
+
+**Is it a hardware wallet?**
+
+<!-- How is it used / how does it connect (e.g. USB)? What software wallets does it integrate with (e.g. MetaMask)? -->
+
+### Security
+
+**Is the wallet code open-sourced?**
+
+<!-- If yes, please provide a direct link to the repository. -->
+
+**What license was the wallet software released under?**
+
+<!-- Please provide information on the software license used for the wallet. -->
+
+**Is your wallet custodial, or non-custodial?**
```

Comment:
```suggestion
**Is your wallet custodial or non-custodial?**
```

Reply:
Is this always binary answer? e.g. didn't Coinbase Wallet recently describe theirs as "semi-custodial"? 
https://twitter.com/petejkim/status/1527027598898905088?s=20&t=9bLBC4t7psrIr-3SNxrFIA

Reply:
Yeah.. Agree entirely with that confusion you mention @minimalsm

Also, quick soap box here related to these terms... I would personally be in favor of ditching both of these terms (Custodial and Non-Custodial).

"Non-custodial wallet" => "wallet"
"Custodial wallet" => "account" (aka not a wallet)

Traditional wallets hold cash... anyone ever refer to their cash wallet as a non-custodial wallet?
Crypto wallets hold keys... your keys, your crypto. Not your keys, not your crypto, and not a wallet.

I push this because over time the idea of "custodial wallets" has become the norm by many regulators, and any form of you holding your own keys to your own funds is seen as abnormal, when in reality it should be flipped. Crypto was founded on individuals holding their own keys, not the other way around. 

Semi-custodial options like the one mentioned I need to read more about myself, but in the meantime I think if the user has no access and complete reliance on a company holding their keys, then it shouldn't be listed as a wallet at all. 

maybe something like..
```suggestion
**Who holds the private keys?**

 <!-- Users must have access to private keys without reliance on a company remaining in business to be listed as a wallet -->
```